drivers/scheduler/k8s/k8s.go

```diff
@@ -166,6 +173,13 @@ func (k *K8s) Init(schedOpts scheduler.InitOptions) error {
 	if err != nil {
 		return err
 	}
+
+	go func() {
+		err = k.collectEvents()
```

Comment:
nit: use `err :=` so that the goroutine doesn't share the `err` variable from the parent function

Reply:
right